prometheus_client/multiprocess.py

```diff
@@ -51,17 +51,20 @@ def merge(files, accumulate=True):
             pid = parts[2] if multiprocess_mode and len(parts) > 2 else None
             try:
                 d = MmapedDict(f, read_mode=True)
-            except FileNotFoundError:
+            except EnvironmentError:
```

Comment:
EnvironmentError being both python2 and python3 compatible, as opposed to FileNotFoundError and IOError